### ISSUE TITLE
Add Optional Capped Collection to Setup

### ISSUE THREAD

**Issue by Barbarrosa** [leebradley/wikiedit-monitor#9]
Let's add these queries to the `init_mongodb.js` script, with configurable sizes and an option to disable/enable the capped option.

``` javascript
db.createCollection('wikiedits', {capped:true, size:100000000000});
db.createCollection('errorlog', {capped:true, size:100000000000});
db.createCollection('socketdata', {capped:true, size:100000000000});
```

You can [read more about capped collections here](https://docs.mongodb.org/manual/core/capped-collections/).

**Comment by Barbarrosa** [comment]
This will be addressed in #17.

**Comment by Barbarrosa** [comment]
This issue has been resolved.
